contracts/src/utils/structs/bitmap.rs

```diff
@@ -0,0 +1,119 @@
+//! Contract module for managing uint256 to bool mapping in a compact and
+//! efficient way, provided the keys are sequential. Largely inspired by Uniswap's <https://github.com/Uniswap/merkle-distributor/blob/master/contracts/MerkleDistributor.sol>[merkle-distributor].
+//!
+//! `BitMap` packs 256 booleans across each bit of a single 256-bit slot of
+//! `uint256` type. Hence, booleans corresponding to 256 _sequential_ indices
+//! would only consume a single slot, unlike the regular `bool` which would
```

Comment:
Here you have `'bool'`, in other places you have just `bool`

Reply:
left just "boolean" word